pkg/kubelet/util/manager/cache_based_manager.go

```diff
@@ -106,10 +108,10 @@ func (s *objectStore) AddReference(namespace, name string) {
 		}
 		s.items[key] = item
 	}
-
-	item.refCount++
 	// This will trigger fetch on the next Get() operation.
-	item.data = nil
+	item.refCount++
+	item.refVersion++
```

Comment:
Why does adding a reference bump the refVersion?

Reply:
Maybe I'm confused about what refVersion represents. I assumed this was a version of the cached data?

In either case, please add a comment to the field explaining what it is.